shuffle/cli/src/account.rs

```diff
@@ -174,6 +180,27 @@ pub fn create_account_onchain(
     Ok(())
 }
 
+async fn create_account_via_faucet(network: &Network, account: &LocalAccount) -> Result<()> {
+    println!("Creating a new account on {}...", network.get_name());
+    let faucet_client = FaucetClient::new(
+        network.get_faucet_url().to_string(),
+        network.get_json_rpc_url().to_string(),
+    );
+
+    let auth_key = account.authentication_key();
+    tokio::task::spawn_blocking(move || {
+        faucet_client.create_account(auth_key, "XUS").unwrap();
```

Comment:
Double checking, do we still need this unwrap? I'm not sure how the unwrap and Result bubble up through spawn_blocking. It's an esoteric question for sure.

What's best practice @bmwill ?

Reply:
![image](https://user-images.githubusercontent.com/55404786/142304188-0a647317-b366-4d37-a220-a0e09af8b370.png)

We need an unwrap() there.

Reply:
you can remove the unwrap and then a Result will be returned from the call to spawn_blocking, well from the future that will resolve when its done so you can bubble up the result.

```
spawn_blocking(|| /* some code */).await??;
```

Reply:
fixed!